modules/exploits/linux/local/pihole_remove_commands_lpe.rb

```diff
@@ -67,7 +67,7 @@ def sudo_pihole
   end
```

Comment:
@sjanusz-r7 Does this need `-n` too? Obviously not for the `check` method, but holistically 👀

Reply:
Good spot 💯  I didn't notice that initially. Fixed now 👍